Library/Formula/safe-rm.rb

```diff
@@ -4,6 +4,8 @@ class SafeRm < Formula
   homepage 'https://launchpad.net/safe-rm'
   url 'https://launchpad.net/safe-rm/trunk/0.10/+download/safe-rm-0.10.tar.gz'
   sha1 '6b829ae68e1fa3c8016e15ab37fcc08caef7712f'
+  
```

Comment:
There's whitespace on this line apparently. You'll need to remove it to pass the bot :).